backend/src/test/java/com/woowacourse/gongcheck/core/application/JobServiceTest.java

```diff
@@ -65,272 +71,598 @@ class JobServiceTest {
+            }
+        }
 
-        assertThat(savedJobId).isNotNull();
+        @Nested
+        class 다른_Host의_Space의_Job_목록을_조회할_경우 {
+
+            private Host host;
+            private Space space;
+
+            @BeforeEach
+            void setUp() {
+                host = hostRepository.save(Host_생성("1234", 1234L));
+                Host otherHost = hostRepository.save(Host_생성("1234", 2345L));
+                space = spaceRepository.save(Space_생성(otherHost, "잠실"));
+            }
+
+            @Test
+            void 예외를_발생시킨다() {
+                assertThatThrownBy(() -> jobService.findJobs(host.getId(), space.getId()))
+                        .isInstanceOf(NotFoundException.class)
+                        .hasMessage("존재하지 않는 공간입니다.");
+            }
+        }
     }
 
-    @Test
-    void Host가_존재하지_않는데_Job_생성_시_예외가_발생한다() {
-        Host host = hostRepository.save(Host_생성("1234", 1234L));
-        Space space = spaceRepository.save(Space_생성(host, "잠실"));
+    @Nested
+    class createJob_메소드는 {
+
+        @Nested
+        class Job_Section들과_Task들을_입력_받는_경우 {
+
+            private Host host;
+            private Space space;
+            private JobCreateRequest request;
+            private List<String> taskCreateRequestNames;
+            private List<String> sectionCreateRequestNames;
+
+            @BeforeEach
+            void setUp() {
+                host = hostRepository.save(Host_생성("1234", 1234L));
+                space = spaceRepository.save(Space_생성(host, "잠실"));
+                List<TaskCreateRequest> tasks = List.of(new TaskCreateRequest("책상 닦기"), new TaskCreateRequest("칠판 닦기"));
+                List<SectionCreateRequest> sections = List.of(new SectionCreateRequest("대강의실", tasks));
+                sectionCreateRequestNames = sections.stream()
+                        .map(SectionCreateRequest::getName)
+                        .collect(Collectors.toList());
+                taskCreateRequestNames = tasks.stream()
+                        .map(TaskCreateRequest::getName)
+                        .collect(Collectors.toList());
+                request = new JobCreateRequest("청소", sections);
+            }
+
+            @Test
+            void 한_번에_생성한다() {
+                Long savedJobId = jobService.createJob(host.getId(), space.getId(), request);
+
+                Job savedJob = jobRepository.getById(savedJobId);
+                List<Section> savedSections = sectionRepository.findAllByJob(savedJob);
+                List<Task> savedTasks = taskRepository.findAllBySectionIn(savedSections);
+
+                assertAll(
+                        () -> assertThat(savedJob.getId()).isEqualTo(savedJobId),
+                        () -> assertThat(savedTasks).extracting(Task::getName)
+                                .containsAll(taskCreateRequestNames),
+                        () -> assertThat(savedSections).extracting(Section::getName)
+                                .containsAll(sectionCreateRequestNames)
```

Comment:
```suggestion
                        () -> assertThat(savedTasks)
                                .extracting(Task::getName)
                                .containsAll(taskCreateRequestNames),
                        () -> assertThat(savedSections)
                                .extracting(Section::getName)
                                .containsAll(sectionCreateRequestNames)
```
개인적으로 띄우는게 낫다라는 의견입니다~